lib/fetch.js

```diff
@@ -15,74 +20,67 @@ export function mapDate(timestamp) {
 }
 
 export function mapLinkList(linkList) {
-  if (linkList) {
-    return linkList.map((link) => {
-      const linkItem = {};
-      if (!link.hasOwnProperty('label') || !link.hasOwnProperty('link')) {
-        return undefined;
-      }
-      linkItem.title = (link.label && link.label.hasOwnProperty('value')) ?
-        link.label.value : '';
-      linkItem.url = (link.link && link.link.hasOwnProperty('value')) ?
-        link.link.value : '';
-      return linkItem;
-    }).filter(l => l !== undefined);
-  }
-  return [];
+  if (!linkList) { return []; }
+  return linkList.map((link) => {
+    if (!link.hasOwnProperty('label') || !link.hasOwnProperty('link')) {
+      return undefined;
+    }
+    const linkItem = {};
+    linkItem.title = pathOr('', ['label', 'value'], link);
```

Comment:
Kinda feels like using library features because they're there when there's only 2, but I don't have a strong opinion either way. Do you have a preference?